bedrock/firefox/templates/firefox/whatsnew/base_142.html

```diff
@@ -0,0 +1,137 @@
+        </button>
+        {% endif %}
+      </div>
+    </header>
+    {% endblock %}
+
+    <div id="outer-wrapper">
+      {% block wnp_content %}
+      <main class="wnp-content">
+        <div class="wnp-content-inner">
+          <h1>What's New in Firefox {{ major_version }}</h1>
+        </div>
+      </main>
+      {% endblock %}
+
+      {% block wnp_footer %}
+      <footer class="wnp-footer">
+        <div class="wnp-footer-inner">
+          <div class="wnp-footer-top">
+            <p class="wnp-footer-copy">
+              Visit <a class="wnp-footer-link" href="https://www.mozilla.org/en-US/">Mozilla Corporation’s</a> not-for-profit parent, the
+              <a class="wnp-footer-link" href="https://foundation.mozilla.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer">Mozilla Foundation</a>.<br>
+              Portions of this content are ©1998–2025 by individual mozilla.org contributors.<br>
+              Content available under a <a class="wnp-footer-link" href="https://www.mozilla.org/en-US/foundation/licensing/website-content/">Creative Commons license</a>.
+            </p>
+            <nav class="wnp-footer-links" aria-label="Footer">
+              <ul>
+                <li>
+                  <a class="wnp-footer-link" href="https://support.mozilla.org/">Firefox Support
+                    <svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 13" width="12" height="12" aria-hidden="true">
+                      <path d="M6.938.5v1.125h3.141L5.337 6.367l.796.795 4.742-4.742v3.143H12V.5H6.937Z" fill="currentColor"/>
+                      <path d="M7.875 11.375h-6.75v-6.75H4.5V3.5H0v9h9V8H7.875v3.375Z" fill="currentColor"/>
+                    </svg>
+                  </a>
+                </li>
+                <li>
+                  <a class="wnp-footer-link" href="/privacy/websites/">Privacy Notice</a>
+                </li>
+                <li>
+                  <a class="wnp-footer-link" href="/about/legal/terms/mozilla/">Terms of Use</a>
+                </li>
+                <li>
+                  <a class="wnp-footer-link" href="/privacy/websites/cookie-settings/">Cookie Policy</a>
+                </li>
+              </ul>
+            </nav>
+            <div class="wnp-footer-logo" aria-hidden="true"></div>
+          </div>
+
+          <hr class="wnp-footer-divider">
+
+          <div class="wnp-footer-bottom">
+            <div class="wnp-footer-brand">
+              <h3 class="wnp-footer-title">Powered by Mozilla</h3>
+              <p class="wnp-footer-subtitle">Putting people before profits since 1998</p>
+            </div>
+            <div class="wnp-footer-cta">
+              <a class="wnp-button wnp-button-outline" href="https://foundation.mozilla.org/?form=whats-new-product"><img src="/media/img/firefox/whatsnew/whatsnew142/heart_filled.svg" width="12" height="12" alt="" aria-hidden="true">Donate to Mozilla Foundation</a>
```

Comment:
Same with the donation link here, too, that can now point to the new domain directly.